### PR TITLE
Docker with tools

### PR DESCRIPTION
**Pull request by dmika1**
## Pull request description

_Provide description for the PR_

## Checklist
- [ ] I have successful end to end tests run (with & without domain)
- [ ] I have added unit tests (if applicable)
- [ ] I have user documentation (if applicable)

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -46,13 +46,19 @@ EOF
...
       ?)    log "Invalid arguments." "ERROR" ; show_help
       esac
   done
 
   shift $((${OPTIND} - 1))
   UNKNOWN_ARGS="$*"
 
+# Clean up before installation
+if [[ ! -f "${CLEAN_UP_FLAG}" ]]; then
+  bash "${SCRIPT_PATH}/cleanup.sh" -s -t -x -r .
```
Comment (bianchi2) [inline install.sh:59]: This must be a script from dcapt repo?

**Review by bianchi2 — APPROVED** [review]

**Comment by bianchi2** [comment]
LGTM. Disregard a failing test.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Example docker command to run install script executed from root dir of the repository
+# docker run --env-file aws_envs \
+# -v "$PWD/k8s_logs:/data-center-terraform/k8s_logs" \
+# -v "$PWD/logs:/data-center-terraform/logs" \
+# -v "$PWD/dcapt.tfvars:/data-center-terraform/dcapt.tfvars" \
+# -it localtf ./scripts/collect_k8s_logs.sh atlas-jira-946-dock-scale-cluster us-east-2 k8s_logs
```
Comment (bianchi2) [inline Dockerfile:6]: This is a log collection example, perhaps it makes sense to update the comment?

**Comment by jjeongatl** [comment]
Thanks for PR!, Should we add this to e2e testing?

**Review by bianchi2 — APPROVED** [review]

**Pull request merged**